src/components/CreateVmWizard/steps/BasicSettings.js

```diff
@@ -208,6 +208,13 @@ class BasicSettings extends React.Component {
         changes.provisionSource = value
         changes.isoImage = undefined
         changes.templateId = undefined
+        if (changes.provisionSource === 'iso') {
+          changes.operatingSystemId = '0'
+          changes.memory = 1024
+          changes.cpus = 1
+          changes.optimizedFor = 'desktop'
+          changes.cloudInitEnabled = false
+        }
```

Comment:
Please try avoid using constants.

In addition, what about handling the other direction of changing ISO->template? It's less important since selecting the specific template will overwrite those values anyway, but there is no use for not resetting it and it's much more elegant that way IMPOV.

Reply:
> In addition, what about handling the other direction of changing ISO->template? It's less important since selecting the specific template will overwrite those values anyway, but there is no use for not resetting it and it's much more elegant that way IMPOV.

Did you consider handling that?

Reply:
I accepted with your review and I changed my code accordingly,
now the form is changed in both directions, and the values are passed as a props,so it's not hard coded.
there are any other changes I missed?